### PR TITLE
Add docs for the modules

### PR DESCRIPTION
**Pull request by Akasurde**
* Added antsibull-docs config
* Updated tox.ini to handle antsibull-docs
* Updated antsibull-docs requirement in test_requirements.txt

Fixes: #160
Depends-On: https://github.com/ansible-community/antsibull-docs/issues/304
Depends-On: https://github.com/ansible-community/antsibull-docs/issues/303 
Depends-On: https://github.com/ansible-community/antsibull-docs/issues/305

### CODE REVIEW

File: docs/ansible.eda.project_info_module.rst [hunk 1]
```diff
@@ -0,0 +1,228 @@
+
+.. Created with antsibull-docs 2.12.0
+
+ansible.eda.project_info module -- List projects in EDA Controller
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This module is part of the `ansible.eda collection <https://galaxy.ansible.com/ui/repo/published/ansible/eda/>`_ (version 1.4.7).
```
Comment (alinabuzachis) [inline docs/ansible.eda.project_info_module.rst:7]: What is this note about? It's unclear to me. Shouldn't this module be part of the 2.0 collection only?

Comment (Akasurde) [reply docs/ansible.eda.project_info_module.rst:7]: This is autgenerated using antsibull-changelog. Need to check how to skip this line in docs for this collection.

Comment (ssbarnea) [reply docs/ansible.eda.project_info_module.rst:7]: I will try to update antsibull-docs to address the current issues. I raised bugs and linked them in the description. They should be easy to address.

**Comment by Akasurde** [comment]
Let us generate docs at the time of 2.0.0 release.

**Comment by ssbarnea** [comment]
> Let us generate docs at the time of 2.0.0 release.

Sure, nothing against it. Is just that I don't want to merge this change w/o running it on CI.

**Comment by ssbarnea** [comment]
Closing as #276 addressed it.

**Pull request closed without merging**